README.md

```diff
@@ -1,28 +1,167 @@
-### !!! Work In Progress !!!
-
-While were close to our public release, we still have a few things to finalize before the repository is ready for public use. Check back in a few days for more details.
-
----
-
+<!-- omit in toc -->
 # Streaming Reactive Framework (SRF)
 
-The Streaming Reactive Framework (SRF) library (pronounced "surf") is designed to build high throughput, low latency pipelines.
+The **Streaming Reactive Framework** (SRF) library (proununced "surf") is a **reactive, network-aware, flexible, and performance-oriented streaming data framework** that standardizes building modular and reusable pipelines with both C++ and Python​. The goal of SRF is to serve as a common high-performance streaming data layer in which all personas of developers - ranging from Data Scientists to DevOps and Performance Engineers can find value.
 
-Major features include:
- - Built in C++ for performance, with Python bindings for ease of use
- - Code-first pipeline design
+<!-- omit in toc -->
+### Major features and differentiators
+ - Built in C++ for performance, with Python bindings for ease of use and rapid prototyping, with options for maximizing performance
  - Distributed computation with message transfers over RMDA using UCX
- - Dynamic reconfiguration to scale up and out at runtime​. Requires no changes to pipeline configuration
- - Unopinionated data model. Messages of any type can be used in the pipeline
+ - Dynamic reconfiguration to scale up and out at runtime​; requires no changes to pipeline configuration
+ - Unopinionated data model: messages of any type can be used in the pipeline
  - Built from the ground up with asynchronous computation for mitigation of I/O and GPU blocking
- - Automatically handles backpressure and reschedules computation as needed
- - Focused on rapid pipeline development, with options for maximizing performance
+ - Automatically handles backpressure (when the sender is operating faster than the receiver can keep up) and reschedules computation as needed
+
+<!-- omit in toc -->
+### Anatomy of a SRF Pipeline
+
+![SRF Pipeline](docs/imgs/srf_pipeline.jpg)
+
+<!-- omit in toc -->
+## Table of Contents
+- [Installation](#installation)
+  - [Prerequisites](#prerequisites)
+  - [Conda Installation](#conda-installation)
+    - [Python Bindings](#python-bindings)
+    - [C++ Bindings](#c-bindings)
+    - [Full SRF Library](#full-srf-library)
+  - [Source Installation](#source-installation)
+  - [Docker Installation](#docker-installation)
+- [Quickstart Guide](#quickstart-guide)
+- [Contributing](#contributing)
+  - [Third-Party code](#third-party-code)
+
+
+## Installation
+SRF includes both Python and C++ bindings and supports installation via [conda](https://docs.conda.io/en/latest/), Docker, or from source.]
```

Comment:
```suggestion
SRF includes both Python and C++ bindings and supports installation via [conda](https://docs.conda.io/en/latest/), Docker, or from [source](#contributing).
```

Reply:
Shouldn't this instead link down to the source directions later in the README instead of `#contributing`?

Reply:
Actually, thinking about this more, I don't like a link here. The TOC delineates how to install from source, and this statement is merely saying that SRF is available to be installed from source. I do need to fix that trailing bracket though.

Reply:
Probably just the trailing bracket.